src/tests/ftest/util/mpio_utils.py

```diff
@@ -115,44 +109,59 @@ def run_llnl_mpi4py_hdf5(self, basepath, hostfile, pool_uuid, test_repo,
         """
         print("self.mpichinstall: {}".format(self.mpichinstall))
         # environment variables only to be set on client node
-        env_variables = {"CRT_ATTACH_INFO_PATH": "{}/install/tmp/"\
-                             .format(basepath),
-                         "MPI_LIB": '',
-                         "MPIO_USER_PATH": "daos:",
-                         "DAOS_POOL": "{}".format(pool_uuid),
-                         "DAOS_SVCL": "{}".format(0),
-                         "HDF5_PARAPREFIX": "daos:"}
-        # setting attributes
-        cd_cmd = 'cd ' + test_repo
+        env = EnvironmentVariables()
+        env["CRT_ATTACH_INFO_PATH"] = "{}/install/tmp/".format(basepath)
```

Comment:
Same as above for 'CRT_ATTACH_INFO_PATH'

Reply:
Done.

Reply:
Still see the env var

Reply:
Should be gone now.